docs/known_issues.md

```diff
@@ -6,15 +6,29 @@ nav_order: 9
 
 # Known issues
 
-## turbo_frame_tag double rendering or scrambled HTML structure
+## Issues resolved by the optional capture compatibility patch
+
```

Comment:
Maybe we should first define what "these issues" are, in terms of symptoms? (Duplicate content, wrongly-ordered HTML tags...)

Reply:
```suggestion

If you're experiencing issues with duplicated content or malformed HTML output, the capture compatibility patch may resolve these.

```